cmd/expired-authz-purger2/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"time"
+
+	"gopkg.in/go-gorp/gorp.v2"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/features"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/metrics"
+	"github.com/letsencrypt/boulder/sa"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type config struct {
+	ExpiredAuthzPurger2 struct {
+		cmd.DBConfig
+		DebugAddr string
+		Syslog    cmd.SyslogConfig
+		Features  map[string]bool
+
+		GracePeriod  cmd.ConfigDuration
+		BatchSize    int
+		WaitDuration cmd.ConfigDuration
+	}
+}
+
+var deletedStat = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Name: "eap2_authorizations_deleted",
+		Help: "Number of authorizations the EAP2 tool has deleted.",
+	},
+)
+
+func delete(clk clock.Clock, gracePeriod time.Duration, batchSize int, dbMap *gorp.DbMap) (int64, error) {
```

Comment:
```suggestion
func doChunk(clk clock.Clock, gracePeriod time.Duration, batchSize int, dbMap *gorp.DbMap) (int64, error) {
```

`delete` is a builtin, generally best not to override.